.github/workflows/test-check.yaml

```diff
@@ -74,26 +70,16 @@ jobs:
       - name: "⚙️ Install dependencies"
         run: pip3 install .[dev]
       - name: "🔬 Running pytorch tests"
-        run: make test TARGETS=pytorch
+        run: |
```

Comment:
No need for working-dir?

Reply:
the checked out code will already bring you to the root level of the repo